eng/Directory.Build.Data.props

```diff
@@ -52,7 +52,7 @@
     <UpdateSourceOnBuild Condition="'$(UpdateSourceOnBuild)' == ''">$(AZURE_DEV_UPDATESOURCESONBUILD)</UpdateSourceOnBuild>
     <PowerShellExe Condition="'$(PowerShellExe)' == ''">pwsh</PowerShellExe>
     <CoverletOutputFormat Condition="'$(CoverletOutputFormat)' == '' and '$(CollectCoverage)' == 'true'">cobertura</CoverletOutputFormat>
-    <InheritDocEnabled Condition="'$(EnableInheritDoc)' == '' or '$(EnableInheritDoc)' == 'true'">true</InheritDocEnabled>
+    <InheritDocEnabled Condition="'$(EnableInheritDoc)' != 'false'">true</InheritDocEnabled>
```

Comment:
You don't need this property anymore then, do you?

Reply:
I guess not. If I set it to true explicitly, would it still respect an msbuild argument to override it to false?